pom.xml

```diff
@@ -425,6 +426,65 @@
         </plugins>
     </build>
 
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
why are you adding in the reporting section this?

Reply:
It is necessary to generate the javadoc report